task/backend/run_recorder.go

```diff
@@ -67,5 +71,5 @@ func (s *StoragePointsWriterRecorder) Record(ctx context.Context, orgID platform
 	}
 
 	// TODO - fix
```

Comment:
Any clues on what this TODO is for, if it was referring to this new addition or something else?